src/index.html

```diff
@@ -1,12 +1,385 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <!--<editor-fold desc="Header">-->
+    <header class="header">
+      <a class="logo" href="#">
+        <img
+          class="logo__img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active is-active">APPLE</a>
+            <!--class is-active only for test-->
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#"
+               class="nav__link"
+               data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--</editor-fold>-->
+
+    <main class="catalog page__catalog">
+      <div class="catalog__wrapper">
+        <!--<editor-fold desc="Card">-->
+        <div class="product-card catalog__product-card" data-qa="card">
+          <img
+            class="product-card__img"
+            src="images/imac.jpeg"
+            alt="imac">
+          <h2 class="product-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="product-card__cod">Код товара: 195434</p>
+
+          <div class="rating product-card__rating">
+            <div class="rating__stars">
+              <div class="rating__star rating__star--active"></div>
+              <div class="rating__star rating__star--active"></div>
+              <div class="rating__star rating__star--active"></div>
+              <div class="rating__star rating__star--active"></div>
+              <div class="rating__star"></div>
```

Comment:
It is okay and made by BEM, but it will more difficult in future to change amount active stars. You can make cycle if help of SASS

Reply:
Also it would better if we make independent block for stars

Reply:
Thanks!!!